### PR TITLE
Upgrade project to gradle

### PR DESCRIPTION
**Pull request by neph1**
(also fixes missing plugin lib)

I've checked that it runs and that you can connect.
Possible further improvements:

- Minie (requires minor refactoring)
- Gui z-order not correct in all places.

### CODE REVIEW

**Comment by Ali-RS** [comment]
I do not think breaking this down into smaller PRs is required as the changes are straightforward forward IMO.

**Comment by neph1** [comment]
Yeah, it does three things:

- Delete ant project files
- Create gradle project files
- Move all classes into the "gradle" folder structure

It's the "move all classes" refactoring that makes it look complex

**Comment by neph1** [comment]
If no one has any further comments, or see a reason why it shouldn't be merged, I'll merge this, shortly.

**Comment by Ali-RS** [comment]
Please see the reviews, there are 4 pending reviews.

**Comment by neph1** [comment]
Long overdue, I think I've fixed everything in your comments.

File: src/main/java/com/mygame/Main.java
Comment (tonihele) [inline src/main/java/com/mygame/Main.java:1]: Is this file needed?

**Review by tonihele — APPROVED** [review]

**Comment by tonihele** [comment]
@Ali-RS do you any demands? We should get this integrated to help all the new comers :)
